packages/repository/src/entity/factory.test.ts

```diff
@@ -31,26 +24,41 @@ describe("Entity", () => {
       id: "super-unique-id",
       foo: "bar",
     })
+
+    vi.clearAllMocks()
```

Comment:
Czy vi.clearAllMocks() w beforeEachu wyczyści nam mocki po ostatnim teście? :D

Reply:
A musi? 😎

Reply:
Nie musi, przecież mocki nie są sharowane po plikach tylko każdy test file ma swoje, dobrze sobie opdowiedziałem?

Reply:
Dobrze 👍 😄 

Generalnie, każdy test file jest modułem ES - więc jest już izolacja na poziomie scope'u. Po wyjściu z pliku testowego giną te zmienne (mechanizm scope chain). Dwa vitest z natury uruchamia się w trybie wielowątkowym - więc każdy plik trafia do innego workera - to dodatkowy stopień izolacji.